src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Legacy/CSharpCodeParser.cs

```diff
@@ -640,10 +638,7 @@ private CSharpStatementBodySyntax ParseStatementBody(Block block = null)
             EnsureCurrent();
             SpanContext.ChunkGenerator = StatementChunkGenerator.Instance;
             AcceptMarkerTokenIfNecessary();
-            if (!At(SyntaxKind.RightBrace))
-            {
-                editHandler.AutoCompleteString = "}";
-            }
+            acceptCloseBraceAccessor.CanAcceptCloseBrace = !At(SyntaxKind.RightBrace);
```

Comment:
I assume it it's not worth having a `WithXXX` that makes a copy of the builder instead of mutating the existing one?

Reply:
This isn't a builder. This is the delayed initialization step.